jetty-websocket/websocket-jetty-client/src/main/java/org/eclipse/jetty/websocket/client/impl/JettyClientUpgradeRequest.java

```diff
@@ -71,13 +60,8 @@ public JettyClientUpgradeRequest(WebSocketCoreClient coreClient, UpgradeRequest
                 .map(c -> new ExtensionConfig(c.getName(), c.getParameters()))
                 .collect(Collectors.toList()));
 
-            // Copy method from upgradeRequest object
-            if (request.getMethod() != null)
-                method(request.getMethod());
-
-            // Copy version from upgradeRequest object
-            if (request.getHttpVersion() != null)
-                version(HttpVersion.fromString(request.getHttpVersion()));
+            // Copy timeout from upgradeRequest object
```

Comment:
No need for the comment.

Reply:
All the other operations in this block have a similar comment. I was trying to fit in with the current style.
Should I remove all these comments to make it more compact?

Reply:
@lachlan-roberts if the comment just says what the code does, remove it.
This one was also rotten, as it was referring to an `upgradeRequest object` which does not exist in the code.